experimental/fs/fs_test.go

```diff
@@ -0,0 +1,51 @@
+package fs_test
+
+import (
+	"context"
+	_ "embed"
+	"log"
+	"os"
+	"testing/fstest"
+
+	"github.com/tetratelabs/wazero"
+	"github.com/tetratelabs/wazero/experimental/fs"
+	"github.com/tetratelabs/wazero/wasi"
+)
+
+// catWasm was compiled the TinyGo source testdata/cat.go
+//go:embed testdata/cat.wasm
+var catWasm []byte
+
+// This is a very basic integration of fs config. The main goal is to show how it is configured.
+func Example_fs() {
```

Comment:
I think this test case overlaps with the examples/wasi/** so maybe I would drop this and simply do unit-test WithFS function something like :
```
func Test_WithFS(t *testing.T) {
	testFs := ....
	ctx, api, err := fs.WithFS(context.Background(), testFs)
	// ...
	
	v := ctx.Value(fs.FSKey{})
	require.NotNil(t, v)
	fsCtx, ok := v.(*wasm.FSContext)
	require.True(t, ok)
	fsCtx.OpenedFile()
	// ...
}
```

the reason is that maybe we want to limit the places where tinygo is used in examples/ directory. @codefromthecrypt wdyt

Reply:
I'd like to offer my favor card by doing this part post merge ;)

Reply:
hah that's good so good as-is then!

Reply:
Actually, I had the same feeling. I just tried to be aligned with experimental/sys_test.go. Please let me fix it.

Reply:
Done https://github.com/tetratelabs/wazero/pull/571/commits/849e48294cfc82898b92e33684f983f9cf71c935